search/search-client-query-transformer/src/main/java/io/camunda/search/clients/DocumentBasedSearchClients.java

```diff
@@ -345,6 +345,12 @@ public SearchQueryResult<GroupEntity> searchGroups(final GroupQuery groupQuery)
         .search(query, io.camunda.webapps.schema.entities.usermanagement.GroupEntity.class);
   }
 
+  @Override
+  public SearchQueryResult<GroupMemberEntity> searchGroupMembers(final GroupQuery filter) {
```

Comment:
:upside_down_face: Parameter should be called `query` instead. In the interface too.